.gitignore

```diff
@@ -1,5 +1,6 @@
-*.DS_Store
-/vendor/
-docs/_build/*
+/build
+/doc/_build
+/vendor
 composer.lock
+phpunit.xml
 .php_cs.cache
```

Comment:
Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore